src/main/java/stockservice/BasicStockService.java

```diff
@@ -46,6 +48,61 @@ public StockQuote getQuote(String symbol) {
         }
         return quoteList;
     }
+    /** Get a historical list of stock quotes for the
+     * provide symbol
+     * @param symbol the stock symbol to search for
+     * @param from the date of the first stock quote
+     * @param interval the number of StockQuotes to get. E.g. if Interval.DAILY was specified
+     *                one StockQuote per day will be returned.
+     * @return a list of StockQuote instances. One for each day in the range specified.
+     */
+    @Override
+    public List<StockQuote> getQuote(@NotNull String symbol, @NotNull Calendar from, @NotNull Calendar until, @NotNull IntervalEnum interval) {
+        StockQuote quote = new StockQuote(Calendar.getInstance(), new BigDecimal("342.1"), "APPL");
+        List<StockQuote> quoteList = new ArrayList<StockQuote>();
+        switch (interval) {
+            case DAILY:
+                for (int doy = from.get(Calendar.DAY_OF_YEAR); doy <= until.get(Calendar.DAY_OF_YEAR); doy++) {
+                    if (symbol == "APPL" && from.before(quote.getDateRecorded()) && until.after(quote.getDateRecorded())) {
+                        quoteList.add(quote);
+                    } else {
+                        throw new MissingResourceException("Record not found", "BasicStockService", symbol);
+                    }
+                }
+                return quoteList;
+
+            case WEEKLY:
+                for (int woy = from.get(Calendar.WEEK_OF_YEAR); woy <= until.get(Calendar.WEEK_OF_YEAR); woy++) {
+                    if (symbol == "APPL" && from.before(quote.getDateRecorded()) && until.after(quote.getDateRecorded())) {
+                        quoteList.add(quote);
```

Comment:
not sure why you are only getting APPL stocks- any symbol should work. 
also use whole world variable names.